src/main/kotlin/lotto/model/LottoGenerator.kt

```diff
@@ -0,0 +1,29 @@
+package lotto.model
+
+import camp.nextstep.edu.missionutils.*
+import lotto.constant.AMOUNT_UNIT
+import lotto.constant.LOTTO_SIZE
+import lotto.constant.MAX_LOTTO_NUM
+import lotto.constant.MIN_LOTTO_NUM
+
+//구매한 로또를 생성하는 클래스
+class LottoGenerator {
+
+    fun generateLotto(money: Int): List<List<Int>> {
+        val randomLotto: MutableList<List<Int>> = mutableListOf()
+        val numberOfLotto = getNumberOfLotto(money)
+
+        for (i in 0 until numberOfLotto) {
+            randomLotto.add(getRandomNumbers())
+        }
```

Comment:
```suggestion
        val randomLotto: MutableList<List<Int>> = mutableListOf()
        val numberOfLotto = getNumberOfLotto(money)

        repeat(numberOfLotto) {
            randomLotto.add(getRandomNumbers())
        }
```
특정 횟수를 반복하려면 for문 대신 repeat를 사용하는게 직관적이며

```suggestion
        val randomLotto = List(getNumberOfLotto(money)) { getRandomNumbers() }
```
또는 List를 특정 횟수만큼 초기화하여 코드 한줄로도 해결이 가능합니다.
덧붙여서 MutableList를 사용하지 않기 때문에 성능이 조금 개선될것 같아요!